tests/test_cli.py

```diff
@@ -161,3 +163,49 @@ def test_all_fields_toml_out(mocker, capsys, make_dist):
             checksum=checksum
         )
     )
+
+
+def test_fmt_ignore_errors_on_stdout(mocker, capsys, make_dist):
+    make_dist(name="distA", install_requires=["distB1>=0.1", "distB2>=0.1"], version="0.1")
+    make_dist(name="distB1", install_requires=["distC1>=0.1", "distC2>=0.1"], version="0.1")
+    make_dist(name="distC2", version="0.1")
+    make_dist(name="distB2", version="0.1")
+
+    mocker.patch(
+        "sys.argv", "johnnydep distA --ignore-errors --fields name".split()
+    )
+    main()
+    out, err = capsys.readouterr()
+    assert err == ""
+    assert out == dedent(
+        """\
+        name
+        ----------------------------
+        distA
+        ├── distB1>=0.1
+        │   ├── distC1>=0.1 (FAILED)
+        │   └── distC2>=0.1
+        └── distB2>=0.1
+    """)
+
+
+@pytest.mark.noautofixt
```

Comment:
This is an unknown marker and creates warnings:

```
tests/test_cli.py:192
  /Users/runner/work/johnnydep/johnnydep/tests/test_cli.py:192: PytestUnknownMarkWarning: Unknown pytest.mark.noautofixt - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.noautofixt
```

It might be better just to patch again on top of the auto-use fixture, and remove the conditional checking for "noautofixt". I'm pretty sure it works fine to patch on a patch.